packages/edit-post/src/components/header/fullscreen-mode-close/index.js

```diff
@@ -11,22 +11,28 @@ import { IconButton, Toolbar } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { addQueryArgs } from '@wordpress/url';
 
-function FullscreenModeClose( { isActive, postType } ) {
-	if ( ! isActive || ! postType ) {
+function FullscreenModeClose( { isActive, postType, isAlwaysShown, isLabeled } ) {
+	if ( ! ( isActive || isAlwaysShown ) || ! postType ) {
 		return null;
 	}
 
+	const label = get(
+		postType,
+		[ 'labels', 'view_items' ],
+		__( 'Back' )
+	);
+
+	const className = isAlwaysShown ? 'edit-post-always-show-close__toolbar' : 'edit-post-fullscreen-mode-close__toolbar';
```

Comment:
We need to follow the naming conventions for modifiers: https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#naming.

Reply:
This extra classname has been removed as this PR has been simplified.  I see this PR is still flagged as "changes requested" though.

Reply:
I guess GitHub doesn't update it until you make another review.